provider/server/server_suite_test.go

```diff
@@ -28,13 +28,16 @@ import (
 )
 
 const (
-	eventuallyTimeout    = 30 * time.Second
-	pollingInterval      = 50 * time.Millisecond
-	consistentlyDuration = 1 * time.Second
-	machineClassx3xlarge = "x3-xlarge"
-	machineClassx2medium = "x2-medium"
-	baseURL              = "http://localhost:8080"
-	streamingAddress     = "127.0.0.1:20251"
+	eventuallyTimeout       = 80 * time.Second
+	pollingInterval         = 50 * time.Millisecond
+	gracefulShutdownTimeout = 60 * time.Second
+	consistentlyDuration    = 1 * time.Second
+	machineClassx3xlarge    = "x3-xlarge"
+	machineClassx2medium    = "x2-medium"
+	squashfsOsImage         = "ghcr.io/ironcore-dev/ironcore-image/gardenlinux:squashfs-dev-20240123-v2"
```

Comment:
acronyms in golang are written with uppercase: squashfsOSImage
More info here: https://google.github.io/styleguide/go/decisions.html#initialisms